src/dmd/mtype.d

```diff
@@ -7252,3 +7215,49 @@ bool isCopyable(Type t)
     }
     return true;
 }
+
+/***************************************
+ * Computes how a parameter may be returned.
+ * Shrinking the representation is necessary because StorageClass is so wide
+ * Params:
+ *   returnByRef = true if the function returns by ref
+ *   stc = storage class of parameter
+ * Returns:
+ *   value from enum ScopeRef
+ */
+ScopeRef buildScopeRef(bool returnByRef, StorageClass stc) pure nothrow @nogc @safe
```

Comment:
If you're not planning to use them in subsequent PRs, I suggest you make these declarations `private`.

Reply:
but I am!